src/components/Wrapper/index.js

```diff
@@ -1,10 +1,18 @@
 import styled from 'styled-components'
 
 export default styled.div`
-  margin-top: ${({ theme, mTop }) => theme.sizes[mTop] || 0};
-  margin-bottom: ${({ theme, mBottom }) => theme.sizes[mBottom] || 0};
-  margin-left: ${({ theme, mLeft }) => theme.sizes[mLeft] || 0};
-  margin-right: ${({ theme, mRight }) => theme.sizes[mRight] || 0};
+  ${({ theme, margin }) => margin && `margin: ${theme.sizes[margin]}`};
+  ${({ theme, mTop }) => mTop && `margin-top: ${theme.sizes[mTop]}`};
+  ${({ theme, mBottom }) =>
+    mBottom && `margin-bottom: ${theme.sizes[mBottom]}`};
+  ${({ theme, mLeft }) => mLeft && `margin-left: ${theme.sizes[mLeft]}`};
+  ${({ theme, mRight }) => mRight && `margin-right: ${theme.sizes[mRight]}`};
+  ${({ theme, padding }) => padding && `padding: ${theme.sizes[padding]}`};
+  ${({ theme, pTop }) => pTop && `padding-top: ${theme.sizes[pTop]}`};
+  ${({ theme, pBottom }) =>
+    pBottom && `padding-bottom: ${theme.sizes[pBottom]}`};
+  ${({ theme, pLeft }) => pLeft && `padding-left: ${theme.sizes[pLeft]}`};
+  ${({ theme, pRight }) => pRight && `padding-right: ${theme.sizes[pRight]}`};
```

Comment:
😱

Reply:
Maybe I'll replace this `Wrapper` component by a wrapper for each component. WDYT?

Reply:
We talked about this with @Tom-Bonnike, we'll keep the Wrapper component, it seems way more convenient 👍